docs/development/translations.rst

```diff
@@ -0,0 +1,408 @@
+Translations
+============
+
+Quick Start Guide
+-----------------
+
+SecureDrop is a system that lets people share sensitive information
+with investigative journalists anonymously and securely.
+Learn :ref:`more about SecureDrop <what_is_securedrop>`.
+
+SecureDrop is written in English and translated in multiple
+other languages. Translations are managed using **Weblate**,
+a web platform that enables collaborative translation projects.
+
+.. _get_started_using_weblate:
+
+Get started using Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can choose to :ref:`register on Weblate <how_to_register_on_weblate_email>`
+with your email address, or by linking a GitHub account.
+
+You can contribute to any language, but Weblate has some conveniences
+to make it easier to work with your preferred languages.
+Learn :ref:`how to choose your preferred languages on Weblate <how_to_manage_your_preferred_languages_weblate>`.
+
+Our Weblate instance only contains one project, SecureDrop, which has
+two translation components: the main SecureDrop web application (labeled
+"SecureDrop"), and the translations for the desktop icons of the admin
+and journalist workstations used by news organizations (labeled "desktop").
+
+.. figure:: ../images/weblate/project.png
+   :align: center
+   :figwidth: 80%
+   :width: 100%
+
+   Components of the SecureDrop project in Weblate. Click on the image to see it full-size.
+
+Once a string is translated, it must be approved by a :ref:`reviewer <weblate_glossary_reviewer>`
+before being accepted into SecureDrop.
+
+Approved strings can only be modified by :ref:`reviewers <weblate_glossary_reviewer>`.
```

Comment:
No need to link the same word twice on sentence apart, IMO.

Reply:
Indeed. I removed the second occurrence.